modules/nw-aws-load-balancer-operator-considerations.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+// * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+
+:_content-type: REFERENCE
+[id="nw-aws-load-balancer-operator-considerations_{context}"]
+= AWS Load Balancer Operator considerations
+
+Review the following limitations before installing and using the AWS Load Balancer Operator.
+
+* The IP traffic mode only works on AWS Elastic Kubernetes Service (EKS). The AWS Load Balancer Operator disables the IP traffic mode for the AWS Load Balancer Controller and hence, the AWS Load Balancer Controller cannot use the pod readiness gate.
```

Comment:
```suggestion
* The IP traffic mode only works on AWS Elastic Kubernetes Service (EKS). The AWS Load Balancer Operator disables the IP traffic mode for the AWS Load Balancer Controller and hence, the AWS Load Balancer Controller cannot use the pod readiness gate either.
```
Just wanted to highlight that 2 things are not working: ip mode and the readiness gate. To me it sounds a little like we disable the ip mode because we don't want to use the pod readiness. Or it's only me?